jetty-core/jetty-http/src/main/java/org/eclipse/jetty/http/content/HttpContent.java

```diff
@@ -83,6 +86,14 @@ default long getBytesOccupied()
 
     void release();
 
+    /**
+     * A {@link Content.Sink} that exposes a pre-sized buffer pool.
+     */
+    interface Sink extends Content.Sink
+    {
+        ByteBufferPool.Sized getSizedByteBufferPool();
+    }
```

Comment:
I'm a big -1 on this interface.   Firstly associating a pool with a sink has nothing really to do with `HttpContent`. Any `Content.Sink` might ultimately have a buffer associated with it and then be wrapped with something like `BufferedContentSink`, which then has it's own buffers

let me ponder a better solution.....